test/e2e/framework/framework.go

```diff
@@ -0,0 +1,60 @@
+package framework
```

Comment:
I'll assume that all the rest of the framework files are copied from Kubernetes. But note that not all of them have Kubernetes copyright headings.
Maybe add a oneline comment to the files that have been copied showing where from, what version (git hash) and if modified a summary of the changes.